orc8r/protos/tenants.proto

```diff
@@ -35,10 +35,22 @@ message IDAndTenant {
     Tenant tenant = 2;
 }
 
+message GetControlProxyResponse {
+  int64 id = 1;
+  string controlProxy = 2;
+}
+
+message CreateOrUpdateControlProxyRequest {
+  int64 id = 1;
+  string controlProxy = 2;
+}
+
 service TenantsService {
     rpc GetAllTenants (Void) returns (TenantList) {}
     rpc GetTenant (GetTenantRequest) returns (Tenant) {}
     rpc CreateTenant (IDAndTenant) returns (Void) {}
     rpc SetTenant (IDAndTenant) returns (Void) {}
     rpc DeleteTenant (GetTenantRequest) returns (Void) {}
+    rpc GetControlProxy (GetTenantRequest) returns (GetControlProxyResponse) {}
+    rpc CreateOrUpdateControlProxy (CreateOrUpdateControlProxyRequest) returns (Void) {}
```

Comment:
These should both be `rpc <name> (<name>Request) returns (<name>Response) {}`

Reply:
Even the void?

Reply:
Correct -- cause e.g. if we suddenly needed to return something in that return proto, we'd have to add the field to `Void` to remain backwards-compatible

Again, not a big issue for our purposes, but is a good general pattern to follow